lua/lspconfig/server_configurations/grammarly.lua

```diff
@@ -0,0 +1,33 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    filetypes = { 'markdown' },
+    root_dir = util.find_git_ancestor,
+  },
+  docs = {
+    description = [[
+Grammarly in an LSP, using the Grammarly API.
```

Comment:
Usually we put the link to the language server as the first part of the description. "an LSP" dosen't make sense if you expand out the acronym (an language server protocol)? "A language server which provides access to Grammarly analysis via the API"

Reply:
Ok, I switched to your description and added the link in 357fa62c0b21e2e262cc7ae2b5e17aa63da8dc09. The link I gave was the one to the upstream repo though, does that make sense to you also, or would you prefer I make it the link to the Grammarly website or to my fork to avoid confusion?

Reply:
You should link to upstream. Also LSP is the protocol, this is a language server.

Reply:
Step 1 is still extremely unclear to me, please clarify the exact additional features afforded by your branch. It's also a long line, break it into multiple lines.

Reply:
> You should link to upstream. Also LSP is the protocol, this is a language server.

Ah, my bad, it seems I have been misusing that acroymn. And also, sorry, I missed that your prior comment provided a description, I swapped it out for that one.

Reply:
> Step 1 is still extremely unclear to me, please clarify the exact additional features afforded by your branch. It's also a long line, break it into multiple lines.

Ok, let me know if 3c118523d37b379039c256d9a9e8a952291da996 clears things up!